simple_sso/sso_server/models.py

```diff
@@ -3,12 +3,14 @@
 
 from django.conf import settings
 from django.db import models
-from django.utils.deconstruct import deconstructible
+try:
+    from django.utils.deconstruct import deconstructible
+except ImportError:
+    deconstructible = None
```

Comment:
You'd get a shorter patch by just `deconstructible = lambda x: x` and allowing it to no-op decorate things.